ethcore/src/spec/spec.rs

```diff
@@ -300,6 +302,10 @@ impl From<ethjson::spec::Params> for CommonParams {
 				BlockNumber::max_value,
 				Into::into,
 			),
+			eip1283_disable_transition: p.eip1283_disable_transition.map_or_else(
+				BlockNumber::max_value,
```

Comment:
eip 1283 is disabled by default too (eip1283 transition also default to max block size).

Reply:
abut the json tag name, it applies `camelCase` to rust field name so it is indeed eip1283DisableTransition in the json.

Reply:
There is no way to _"set Constantinople"_ in Parity Ethereum. You either enable EIP-1283 or you don't (default). You either disable it or you don't (default). So a default chain does neither have a 1283-transition nor a 1283-disable-transition.

Reply:
> You either enable EIP-1283 or you don't (default)

Not quite true. You either 
1. Define it, as disabled, 
2. Define it, as enabled, 
3. Don't define it. 

In the case where you have set a blocknumber for `EIP-1283`, but haven't _defined_ `eip1283_disable_transition`, it will default use `1283`. Which is what I'm talking about. 

Just clarifying what I meant, not trying to argue that you should do it the way I proposed...

Reply:
I'm not understanding your comment. if you _define_ something, than you are not using _defaults_.

- `eip1283Transition` is default `BlockNumber::max_value`, hence disabled
- `eip1283DisableTransition` is default `BlockNumber::max_value`, hence disabled

So if you don't define 1283, there won't be 1283 by default. You only get it if you set `eip1283Transition` explicitly.

Reply:
Right. So if you have a chain spec which your colleague gave you a month ago, which has `eip1283Transition` defined as (in two wooks), and update to the latest parity client, it will roll out the version with `1283`, because `eip1283_disable_transition` is not defined.
 
An alternative approach would be to have it at `0` unless otherwise defined.